reference/proto-definition.md

```diff
@@ -82,6 +82,11 @@ IS statements may appear inside the PROTO definition wherever fields may appear.
 IS statements shall refer to fields defined in the PROTO declaration. Multiple
 IS statements for the same field in the PROTO interface declaration is valid.
 
+An interface field that is not linked to an internal PROTO field with an IS will
+produce a warning when parsed. This warning can be suppressed when declaring an 
+interface field by using the `unconnectedField` keyword instead of `field` in the
+declaration.
```

Comment:
This is not completely correct, as if the field is used by Lua, then the `unconnectedField` is useless as there is no warning anyway.

But I agree it would be nice to explain in which case this keyword is useful. To main the main use is to use this field for 'storing' values used by a Supervisor.